src/stateManagement/useDataStore.ts

```diff
@@ -3,8 +3,13 @@ import { DatabaseView } from "DatabaseView";
 import create from "zustand";
 import data_state_actions from "stateManagement/data/DataStateActions";
 
+export const __set__: {
```

Comment:
This should only be visible by zustand hooks. Could provokes shadow rerenders on some component.